### PR TITLE
dev, v2.1, v3.0, v3.1/reference: change the description of SQL_MODE

### PR DESCRIPTION
**Pull request by wshwsh12**
### What is changed, added or deleted? 
Fix the description of SQL_MODE.
Now TiDB default SQL mode is same with MySQL 5.7, but different of MySQL 8.0.

### Which version does your change affect? 
dev, v3.0, v3.1, v2.1

### CODE REVIEW

File: dev/reference/mysql-compatibility.md [hunk 1]
```diff
@@ -159,8 +159,11 @@ TiDB 支持 MySQL 5.7 中 **绝大多数的 SQL 模式**，以下几种模式除
 + `foreign_key_checks` 的默认值不同：
     + TiDB 中该值默认为 `OFF`，并且目前 TiDB 只支持设置该值为 `OFF`。
     + MySQL 5.7 中该值默认为 `ON`。
-+ 默认 SQL mode 与 MySQL **已相同**
-    + TiDB 和 MySQL 5.7 中均为 `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
++ 默认 SQL mode 与 MySQL 不同：
+    + TiDB 中为 `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
+    + MySQL 中默认设置：
+        + MySQL 5.7 中与 TiDB 相同
```
Comment (juliezhang1112) [inline dev/reference/mysql-compatibility.md]:
```suggestion
        + MySQL 5.7 的默认 SQL mode 与 TiDB 相同
```

File: v2.1/reference/mysql-compatibility.md [hunk 2]
```diff
@@ -138,8 +138,11 @@ TiDB 支持 MySQL 5.7 中 **绝大多数的 SQL 模式**，以下几种模式除
     + TiDB 中，`utf8mb4` 的默认排序规则为 `utf8mb4_bin`
     + MySQL 5.7 中，`utf8mb4` 的默认排序规则为 `utf8mb4_general_ci`，MySQL 8.0 中修改为 `utf8mb4_0900_ai_ci`
     + 请使用 [`SHOW CHARACTER SET`](/v2.1/reference/sql/statements/show-character-set.md) 语句查看所有字符集的默认排序规则
-+ 默认 SQL mode 与 MySQL **已相同**
-    + TiDB 和 MySQL 5.7 中均为 `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
++ 默认 SQL mode 与 MySQL 不同：
+    + TiDB 中为 `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
+    + MySQL 中默认设置：
+        + MySQL 5.7 中与 TiDB 相同
```
Comment (juliezhang1112) [inline v2.1/reference/mysql-compatibility.md]:
```suggestion
        + MySQL 5.7 的默认 SQL mode 与 TiDB 相同
```

File: v3.0/reference/mysql-compatibility.md [hunk 3]
```diff
@@ -159,8 +159,11 @@ TiDB 支持 MySQL 5.7 中 **绝大多数的 SQL 模式**，以下几种模式除
 + `foreign_key_checks` 的默认值不同：
     + TiDB 中该值默认为 `OFF`，并且目前 TiDB 只支持设置该值为 `OFF`。
     + MySQL 5.7 中该值默认为 `ON`。
-+ 默认 SQL mode 与 MySQL **已相同**
-    + TiDB 和 MySQL 5.7 中均为 `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
++ 默认 SQL mode 与 MySQL 不同：
+    + TiDB 中为 `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
+    + MySQL 中默认设置：
+        + MySQL 5.7 中与 TiDB 相同
```
Comment (juliezhang1112) [inline v3.0/reference/mysql-compatibility.md]:
```suggestion
        + MySQL 5.7 的默认 SQL mode 与 TiDB 相同
```

File: v3.1/reference/mysql-compatibility.md [hunk 4]
```diff
@@ -159,8 +159,11 @@ TiDB 支持 MySQL 5.7 中 **绝大多数的 SQL 模式**，以下几种模式除
 + `foreign_key_checks` 的默认值不同：
     + TiDB 中该值默认为 `OFF`，并且目前 TiDB 只支持设置该值为 `OFF`。
     + MySQL 5.7 中该值默认为 `ON`。
-+ 默认 SQL mode 与 MySQL **已相同**
-    + TiDB 和 MySQL 5.7 中均为 `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
++ 默认 SQL mode 与 MySQL 不同：
+    + TiDB 中为 `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
+    + MySQL 中默认设置：
+        + MySQL 5.7 中与 TiDB 相同
```
Comment (juliezhang1112) [inline v3.1/reference/mysql-compatibility.md]:
```suggestion
        + MySQL 5.7 的默认 SQL mode 与 TiDB 相同
```

File: dev/reference/mysql-compatibility.md [hunk 5]
```diff
@@ -159,8 +159,11 @@ TiDB 支持 MySQL 5.7 中 **绝大多数的 SQL 模式**，以下几种模式除
 + `foreign_key_checks` 的默认值不同：
     + TiDB 中该值默认为 `OFF`，并且目前 TiDB 只支持设置该值为 `OFF`。
     + MySQL 5.7 中该值默认为 `ON`。
-+ 默认 SQL mode 与 MySQL **已相同**
-    + TiDB 和 MySQL 5.7 中均为 `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
++ 默认 SQL mode 与 MySQL 不同：
```
Comment (XuHuaiyu) [inline dev/reference/mysql-compatibility.md]: 与 MySQL 5.7 相同，与 MySQL 8.0 不同

Comment (wshwsh12) [reply dev/reference/mysql-compatibility.md]: Addressed. @XuHuaiyu  PTAL

**Pull request merged**